apps/test/unit/templates/sectionProgress/SectionProgressTest.js

```diff
@@ -68,4 +70,16 @@ describe('SectionProgress', () => {
     );
     expect(wrapper.find('#uitest-detail-view').exists()).to.be.true;
   });
+
+  it('standards view shows standards view and legend', () => {
```

Comment:
the name here seems a little confusing. The legend is part of the standards view right? Should it explicitly check for the legend or is it just checking for the whole standards view?